styleguide/index.html

```diff
@@ -6,7 +6,7 @@
 <!--[if (gte IE 9)|(gt IEMobile 7)]><!--> <html class="no-js" lang="en"> <!--<![endif]-->
 <head>
   <meta charset="utf-8">
-  <title>USEITI Style Guide</title>
+  <title>KSS Style Guide</title>
```

Comment:
To fix this I think we need to make a KSS template? I made a ticket for it here: #769